homework.txt

```diff
@@ -0,0 +1,67 @@
+//the subway lines..
+
+theNline = [Times Square, 34th, 28th, 23rd, Union Square, 8th]
+theLline = [8th, 6th, Union Square, 3rd, 1st]
+the6line = [Grand Central, 33rd, 28th, 23rd, Union Square, Astor Place]
+
+//examples of inputs..
+
+planTrip("theNline", "Times Square", "theNline", "28th")
+planTrip("theNline", "8th", "theNline", "34th")
+planTrip("Times Square", "28th")         //2
+planTrip("8th", "34th")                  //4
+------------------------------------------------------------------
+
+//input variable for theNline
+
+var theNline = ["Times Square", "34th", "28th", "23rd", "Union Square", "8th"]
+------------------------------------------------------------------
+
+//planTrip function for the first line..
+
+function planTrip (onStop, offStop) {
+var onStopIndex,offStopIndex, numOfStops;
+for (i=0 ; i < theNline.length ; i++) {
+if (theNline[i] === onStop) {onStopIndex = i}
+if (theNline[i] === offStop) {offStopIndex = i}
+}

```

Comment:
This is great! This is more or less how `indexOf()` actually works underneath. You can [read more about it here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)